src/main/java/net/pms/formats/OGG.java

```diff
@@ -0,0 +1,27 @@
+package net.pms.formats;
+
+public class OGG extends Format {
+
+	@Override
+	public Identifier getIdentifier() {
+		return Identifier.OGG;
+	}
+
+	@Override
+	public boolean transcodable() {
+		return true;
+	}
+
+	public OGG() {
+		type = VIDEO;
+	}
+
+	@Override
+	public String[] getSupportedExtensions() {
+		return new String[] {
+			"ogg",
+			"ogm",
+			"ogv"
```

Comment:
Sorry, i prefer give up.
That's said it's not your fault, the renderers are mostly bad copy haste that i'm amazed to see used without much issues report.